README.md

```diff
@@ -10,8 +10,9 @@ Following crDroid 8.0 release all rules are enforced.
 
 ### General rules
 * All maintainers must have knowledge about source control tools such as *git* and *repo*.
-* All maintainers must release device sources **publicly** at crDroid Android organization on Github, here includes common device tree if present, device tree, vendor and kernel
-* In the case of incompatibilities with current already hosted device sources, maintainers will need to create a dedicated branch for their devices on said repos
+* All maintainers must release device sources **publicly** at crDroid Android organization on Github, here includes common device tree (if present), device tree, vendor and kernel
+* All sources must be fully synced (pushed to GitHub) **prior to** every official build release
```

Comment:
How to validate?

Reply:
We can't validate any of these points as maintainers are building on their own computers, we just have to trust them :)
Also, from time to time we should check a few devices chosen randomly from ota repository and punish maintainers if needed.